src/workerd/io/compatibility-date.capnp

```diff
@@ -69,6 +69,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   annotation impliedByAfterDate @0xe3e5a63e76284d89 (field) :ImpliedByAfterDate;
 
+  annotation pythonRelease @0xef74c0cc5d18cc0c (field) :Text;
+  # A tag to tell us which compat flags represent a Python Workers release and thus need to be
+  # taken into account for memory snapshot compatibility.
```

Comment:
For memory snapshot compatibility we need much more than this. I think we should hash `pyodide.asm.wasm` and `pyodide.asm.js` and key on that.

Reply:
Won't these files only change across compat flags?

Reply:
Yeah. The Python version will also only change across compat flags though.

Reply:
Renamed the annotation to `pythonSnapshotRelease` and reworded the description to make things clearer.